admin/testadmin/testadmin.go

```diff
@@ -50,6 +53,23 @@ type Fixture struct {
 	ServerOpts *server.Options
 }
 
+// NewGithub creates a new Github client. In short testing mode this is a mock client which has no-op implementations of all methods.
+// Otherwise it creates a real implementation that makes real API calls to Github.
+func NewGithub(t *testing.T) admin.Github {
+	if testing.Short() {
+		return &mockGithub{}
+	}
+	err := godotenv.Load("../../../.env")
+	require.NoError(t, err, "failed to load env file. Run `rill devtool dotenv refresh` to create one")
+
+	githubAppID, err := strconv.ParseInt(os.Getenv("RILL_ADMIN_GITHUB_APP_ID"), 10, 64)
+	require.NoError(t, err)
+
+	github, err := admin.NewGithub(t.Context(), githubAppID, os.Getenv("RILL_ADMIN_GITHUB_APP_PRIVATE_KEY"), os.Getenv("RILL_ADMIN_GITHUB_MANAGED_ACCOUNT"), zap.Must(zap.NewDevelopment()))
+	require.NoError(t, err)
+	return github
```

Comment:
Should we have test-scoped names for these? E.g. `RILL_ADMIN_TEST_GITHUB_APP_PRIVATE_KEY` and `RILL_ADMIN_TEST_GITHUB_MANAGED_ACCOUNT`. 

Just to keep tests configured separately. Maybe I'm wrong, but I don't believe any tests currently use the normal env variables.